lib/cli/watch-run.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const utils = require('../utils');
+const Context = require('../context');
+const Mocha = require('../mocha');
+
+/**
+ * Exports the `watchRun` function that runs mocha in "watch" mode.
+ * @see module:lib/cli/run-helpers
+ * @module
+ * @private
+ */
+
+/**
+ * Run Mocha in "watch" mode
+ * @param {Mocha} mocha - Mocha instance
+ * @param {Object} opts - Options
+ * @param {string[]} [opts.extension] - List of extensions to watch
+ * @param {string|RegExp} [opts.grep] - Grep for test titles
+ * @param {string} [opts.ui] - User interface
+ * @param {string[]} [opts.files] - Array of test files
+ * @param {string} [opts.cwd] - Current working directory
```

Comment:
This new object property `opts.cwd` was previously just a referenced variable that defaulted to `process.cwd()`; at last count, it isn't a Mocha option either. Why bother passing this? Maybe L36 should be:
```js
  const watchFiles = utils.files(process.cwd(), extension);
```

Reply:
Addressed in https://github.com/mochajs/mocha/pull/3930/commits/595b275411d944656ffd3f7874c22bcd77483d1c